From 696f4e970203c227340a49c376879d1efc93b3fb Mon Sep 17 00:00:00 2001
From: "guoyin.chen" <guoyin.chen@freescale.com>
Date: Wed, 4 Sep 2013 10:57:30 +0800
Subject: [PATCH 249/602] ENGR00275504 Encryption : Data Encryption is invalid

Data Encryption function has the precondition with batter status
Add property to make a simulation of battery&charge status

Signed-off-by: guoyin.chen <guoyin.chen@freescale.com>
---
 evk_6sl/init.rc | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/evk_6sl/init.rc b/evk_6sl/init.rc
index 934828a..82f9c3f 100644
--- a/evk_6sl/init.rc
+++ b/evk_6sl/init.rc
@@ -19,6 +19,9 @@ on init
 
 on boot
 
+    # emmulate battery property
+    setprop sys.emulated.battery 1
+
     # Set permission for IIM node
     symlink /dev/mxs_viim /dev/mxc_mem
 
-- 
1.8.0

